README.md

```diff
@@ -1,78 +1,65 @@
+  specified for MariaDB.
+* **Matomo database password**, `MATOMO_DATABASE_PASSWORD=<db-password>`: Same password
+  (non-root) that was specified for MariaDB.
+* **Matomo database name**, `MATOMO_DATABASE_NAME=<db-name>`: Same database name that was
+  specified MariaDB.
 
-### Variables
-
-* **Application name**, `NAME=<app-name>`: Unique identifier for your application,
-  recommended value - your username
-* **Username**, `USERNAME=<app-username>`: Create a new username for logging into Matomo
-* **Password**, `PASSWORD=<app-password>`: Create a new password for logging into Matomo
-* **Application hostname suffix**, `APPLICATION_DOMAIN_SUFFIX=<app-suffix>`: The exposed
-  hostname suffix that will be used to create routes for the application. Default when
-  using Rahti is `rahtiapp.fi`
-* **Matomo database username**, `MATOMO_DATABASE_USER=<db-username>`: Username that was
-  specified when deploying MariaDB in the previous step
-* **Matomo database password**, `MATOMO_DATABASE_PASSWORD=<db-password>`: Password
-  (non-root) that was specified when deploying MariaDB in the previous step
-* **Matomo database name**, `MATOMO_DATABASE_NAME=<db-name>`: Database name that was
-  specified when deploying MariaDB in the previous step
 
 ### Command-line deployment of Matomo
 
 1. Download the `matomo-template.yaml` from from this repo.
 2. Process and apply template using default values from template and passing you application
-   specific parameters.
+   specific parameters, sample example shown below:-
 
 ```bash
-oc process -f matomo-template.yaml -p NAME=<app-name>  -p USER=<app-username> PASSWORD=<app-password>  -p MATOMO_DATABASE_USER=<db-username> -p MATOMO_DATABASE_PASSWORD=<db-password> -p MATOMO_DATABASE_NAME=<db-name> | oc apply -f -
+oc process -f matomo-template.yaml -p MATOMO_APP_NAME="matomo-app-johndoe" -p MYSQL_USER="matomouser" -p MYSQL_PASSWORD="matomo" -p MYSQL_ROOT_PASSWORD="letmeinroot" -p MYSQL_DATABASE="matomodb" -p MATOMO_USERNAME="matomo" -p MATOMO_PASSWORD="matomopass" -p MATOMO_DATABASE_USER="matomouser"  -p  MATOMO_DATABASE_PASSWORD="matomo" -p  MATOMO_DATABASE_NAME="matomodb" | oc apply -f 
 ```
 
 ### Deleting the application and project
 
 ```bash
-oc process -f matomo-template.yaml -p NAME=<app-name>  -p USER=<app-username> PASSWORD=<app-password>  -p MATOMO_DATABASE_USER=<db-username> -p MATOMO_DATABASE_PASSWORD=<db-password> -p MATOMO_DATABASE_NAME=<db-name> | oc delete -f -
+oc process -f matomo-template.yaml -p MATOMO_APP_NAME="matomo-app-johndoe" -p MYSQL_USER="matomouser" -p MYSQL_PASSWORD="matomo" -p MYSQL_ROOT_PASSWORD="letmeinroot" -p MYSQL_DATABASE="matomodb" -p MATOMO_USERNAME="matomo" -p MATOMO_PASSWORD="matomopass" -p MATOMO_DATABASE_USER="matomouser"  -p  MATOMO_DATABASE_PASSWORD="matomo" -p  MATOMO_DATABASE_NAME="matomodb" | oc delete -f -
 ```
 
+or 
+
 * `oc delete all -l app=<app-name>`
 * `oc delete secret -l app=<app-name>,deploymentconfig=<app-name>,template=<app-name>`
   
```

Comment:
I think it's good to write `matomo` instead of `<app-name>` in this example:  
```sh
oc delete all -l app=matomo 
oc delete secret -l app=matomo
oc delete pvc -l app=matomo
```